EiDiA_frontend/src/components/exportView/subcomponents/SaveTemplateWindow.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {Button, Input, Typography} from '@material-ui/core';
+import {Column, Row} from '../../../support files/constants';
+
+const styles = {
+    button_left: {
+        margin: "15px",
+        align: "left",
+        bottom: 0,
+    },
+    button_right: {
+        margin: "15px",
+        align: "right",
+        bottom: 0,
+        right: 0,
+    }
+}
+
+export default class SaveTemplateSection extends React.Component {
+    render() {
+        return (
+            <Column>
+                <Row>
+                    <Typography variant="subtitle2">
+                        Template Name
+                    </Typography>
+                </Row>
+                <Row>
+                    <Input placeholder="Template Name" inputProps={{'aria-label': 'description'}}/>
```

Comment:
```suggestion
                    <Input placeholder="Template Name"/>
```
`description` ist keine Beschreibung